ch-covidcertificate-backend-verifier/ch-covidcertificate-backend-verifier-sync/src/main/java/ch/admin/bag/covidcertificate/backend/verifier/sync/config/SyncCloudBaseConfig.java

```diff
@@ -31,28 +33,26 @@ public abstract class SyncCloudBaseConfig extends SyncBaseConfig {
 
     @Value("${datasource.leakDetectionThreshold:0}")
```

Comment:
this property (as well as `dataSrouceMaximumPoolSize` and `dataSrouceConnectionTimeout`) is not being applied to the datasource config anymore. is that ok? if yes delete those fields in this config as they are unused